scripts/core/ui/components/prompt-for-unsaved-changes.tsx

```diff
@@ -62,6 +62,7 @@ export function showUnsavedChangesPrompt(
                         highlightOption: true,
                     },
                 ],
+                'options-modal'
```

Comment:
data-test-id here should be `unsaved-chaged-dialog` or something similar. Options modal is a very generic name.